app/src/main/res/drawable-night/ic_authorizations_empty.xml

```diff
@@ -0,0 +1,135 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please add `Copyright`

Reply:
Can we use it this file for both themes?

Reply:
in some places there are single resource for both themes